imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml

```diff
@@ -36,6 +36,28 @@ energy_attrs:
     VALIDMAX: 127
     VAR_TYPE: support_data
 
+inst_az_attrs:
+    <<: *default
+    CATDESC: Azimuth
+    FIELDNAM: Azimuth
+    FORMAT: I2
+    LABLAXIS: Azimuth
```

Comment:
I'm planning to provide better labels/descriptions for these in an upcoming PR where I address nomenclature